tests/integration/common-operator-integ-suite.sh

```diff
@@ -158,6 +158,11 @@ check_ready() {
   ${COMMAND} wait deployment "${DEPLOYMENT_NAME}" -n "${NS}" --for condition=Available=True --timeout=${TIMEOUT}
 }
 
+fail() {
```

Comment:
nit: When I read a function called `fail` I expect it to fail (i.e., `exit 1`) which is not the case here. It's confusing.

Reply:
renamed to `logFailure`